technical-doc/api.rst

```diff
@@ -879,307 +1442,1820 @@ associated to a given hash(s). - Query:
-                                "2AamJsrQ68r3r9bhgsiAKkH7JGUSDLPCZUh91DxeHTBQWBRjQu",
-                                "2LSHyysq3PRDhfYF68haUWHUW7oAwZXhzqewLJhAmVRwH6JjVh",
-                                "jWvGUobun7mzuaJye5nYEFvSGrmyW4sjrmEZ3mvTAo7iQtmxQ",
-                                "JsfRiXaoZY4tDqJWyEEpJkjchrcrGpdi2im77KW9bcGjX8FrW",
-                                "FCzk7rYB8ZAqkkSchdxPeZQohjNU4Wwi2TdvNJe1Df14LnxW4",
-                                "2HrJiz9fBDVUjVTF6aA3y8bZQcHVpnJygRkt8EtnkCB3HT9dxM",
-                                "242ghWYwu8PKZSQoPoij2S8CE2u46zfcGdC42mNN12HFrj55aQ",
-                                "2UCUqS26mbdbsFZkL1rNrrQu2SMZQnUZdTJ7tjo7QVjVVFvfxR",
-                                "2Ch6hGDfEeUBamUrWojzrTPeswTuGVfwvjfDEKChNHU96A3PXo",
-                                "pyK7qz9ebZ9bModjmuvvUoVmvBrLHUsUy8uqWRKemTh2zLFyd",
-                                "2SJhPnDC8nV4SkaxTai4Gvxpzv14DeD164XHqKfSbV5byTjfqq",
-                                "jsZEsz4U3jazbpLfXvK6BVRKyE4F3Sh6bS26AN9i7vM274XQc",
-                                "2K75STq46JPJ2eUzZrepRNDPrfg6NgKC9cZFYFVXfKpJVBQAP8",
-                                "3QYdeQrNsbBXpEUbqBY1v1UPWiwnvgSjL2mcG1fzkRC4Mso5J",
-                                "V58LdmjJLvQLRVm4bqqiMn2ChGprgLDxwNm4gKJqz3UXFwonK",
-                                "AvvHCpxnX8U6uTQKmpze55vzhkhdbbst9rrhPwbykymjJyzoV"
-                            ],
-                            "slot": {
-                                "period": 72180,
-                                "thread": 31
-                            }
+                            ]
                         },
-                        "creator_address": "A12Lh4yCTdF5P9Unt3Aq4aimUFafTiuGoYYHfqU2P68RkY35tFmQ",
-                        "creator_public_key": "P12uRiEAgEGHPpNZ7fMFsh3XPYS6A9GB9PgDAQdwFbxCH4nxSQEF",
-                        "id": "Fb46NHJCFTVgddSEZMEcmYeYpokvQv8gCYjnDBpXbAQBKpVE3",
-                        "signature": "KMcsEkC6CEDE55tmvAfR47vhH1PvAd52SZWUyQ9kdobvkvEGMZS2TrDYHWP6jCod1QeQpaRY1BXM2FMSWn5EuCS5HcVti"
+                        "signature": "4vKvfytWkgv5YjTQQiVwADpBMpvAF6W79KBHzAs1iybLswUZ2VhWPh6dMTnifVeiBYXVJuvgcMkKkRH8VjL2izkRVTAxs",
+                        "content_creator_pub_key": "P1snABBASHQQuCDERNdUBzg2kJ37LLFK8L7sgBoBd32mRU38ev9",
+                        "content_creator_address": "AU1WqP6GTsR8w4eyRPHQkrQhujmRb3hmLaQt3HZfeTVHWVZade48",
+                        "id": "B122ByHzPVJ3QFwmuYcZ4vZYzq6rfkqx7BJSJdFNHWp9j2o5Fpxv"
                     },
                     "operations": [
-                        "177bzpUmukLarBiRGcTCDE63xqc5nkAKUja414HDmsNS2T3Gy",
-                        "1G9oj5pX7Ruj1w6cT9DXY2KMNKM9JgHVfA6JN5nhZchnHCm6s",
-                        ...
-                        "2wX8fv5WaPpEXFYPbzmyeShVMRNdcqGCxnUCdfXKW2LrVvpkM1",
-                        "2wdwXBXxpBeQuPtT5kkEiV3AKsTqWkzN9QLHXskDsZgD8XpdTN"
+                        "O116jye7zBAttWHfCufcMVQEQaa9JMTqv6To5mE2CzmzBs7cZ7W",
```

Comment:
There's way too much information. This makes the page gigantic and unusable.

Reply:
done